consoles/VNC.pm

```diff
@@ -759,7 +759,7 @@ sub map_and_send_key {
     }
     usleep(2_000);
     if (!defined $down_flag || $down_flag == 0) {
-        for my $key (@events) {
+        for my $key (reverse @events) {
```

Comment:
I guess a comment here would really be a good idea explaining the delicate impact of "reverse" :)

Reply:
git blame will help you if you're super curious

Reply:
...until 'tidy' hits the line, or something...

so yeah, I'm with @okurz (in fact I meant to add a comment originally but forgot, so thanks for the reminder!)